admin/sql/updates/2018-06-25-drop-artist-credit-redirect.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
```

Comment:
Not sure this is really needed since we haven't updated production yet -- so that table will not exist there...